static/js/FieldObj.js

```diff
@@ -51,3 +51,19 @@ class InteractableObj extends FieldObj {
         this.direction = dir;
     }
 }
+
+class Ramp extends FieldObj {
+    constructor(x, y, w, h, highSide = "up", incline = 15, color = "black") {
+        super(x, y, w, h, color);
+        if (highSide == "up") {
+            this.highSide = 90.0;
+        } else if (highSide == "down") {
+            this.highSide = 270.0;
+        } else if (highSide == "right") {
+            this.highSide = 0.0;
+        } else if (highSide == "left") {
+            this.highSide = 180.0;
+        }
```

Comment:
Are we using these numerical values anywhere? If not, will they be useful in the future?